sdv/constraints/tabular.py

```diff
@@ -746,32 +761,41 @@ class ScalarRange(Constraint):
             not ``>`` when comparing them.
     """
 
+    @staticmethod
+    def _validate_inputs(low_value, high_value):
```

Comment:
again minor, but I don't think validate methods should return anything. There purpose should just be to raise errors